spectator/__init__.py

```diff
@@ -7,10 +8,15 @@
 
 GlobalRegistry = Registry()
 
+logger = logging.getLogger("spectator.init")
+
 try:
     from spectatorconfig import auto_start_global
+
     if auto_start_global():
         GlobalRegistry.start()
         atexit.register(GlobalRegistry.stop)
-except:
-    pass
+    else:
+        logger.warning("module spectatorconfig auto-start is disabled - GlobalRegistry will not start")
+except ImportError:
+    logger.error("module spectatorconfig is missing - GlobalRegistry will not start")
```

Comment:
The downside to this is that for things like running tests or other uses where you do not need it you will now get an error logged.

Reply:
What about moving it to debug level?

It's really common for people to forget to add the bindings and wonder why nothing works.